chain/client/src/client.rs

```diff
@@ -477,20 +477,19 @@ impl Client {
 
     fn should_reschedule_block(
```

Comment:
While you're at it, can you add some comments please?

Reply:
Will do in following PR.